clients/include/testing_tbmv_strided_batched.hpp

```diff
@@ -103,53 +103,50 @@ void testing_tbmv_strided_batched(const Arguments& arg)
     rocblas_local_handle handle;
 
     // argument sanity check before allocating invalid memory
-    if(M < 0 || K < 0 || lda < M || lda < 1 || !incx || K >= lda || batch_count <= 0)
+    bool invalidSize = M < 0 || K < 0 || lda < K + 1 || !incx || batch_count < 0;
+    if(invalidSize || !M || !batch_count)
     {
```

Comment:
can you use snake_case here rather than camelCase, so invalid_size rather than invalidSize